cmd/influx/v1_authorization.go

```diff
@@ -479,9 +519,78 @@ func v1AuthorizationInactiveF(cmd *cobra.Command, args []string) error {
 	})
 }
 
+var (
+	errMultipleMarchingAuthorizations = errors.New("multiple authorizations found")
```

Comment:
```suggestion
	errMultipleMatchingAuthorizations = errors.New("multiple authorizations found")
```

Reply:
Thanks! I am not used to typing on this laptop keyboard – one of the joys of being in quarantine in Australia right now 🤣